Cargo.toml

```diff
@@ -27,9 +27,10 @@ targets = [
 ]
 
 [features]
-default = [ "tray" ]
-tray = [ "libappindicator" ]
-ayatana = [ "libayatana-appindicator" ]
+default = [ "tray", "ayatana-appindicator" ]
+tray = []
+gtk-appindicator = [ "tray", "libappindicator" ]
+ayatana-appindicator = [ "tray", "libayatana-appindicator" ]
```

Comment:
Could we shorten the name a bit like `gtk-tray` and `ayatana-try`?
The length is getting a bit longer imo.😅

Reply:
makes sense

Reply:
pushed @wusyong